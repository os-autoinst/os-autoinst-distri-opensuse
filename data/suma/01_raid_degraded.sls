partitioning:
    disk1:
        type: DISK
        device: /dev/sda
        disklabel: msdos
        partitions:
            p1:
                 size_MiB: 2000
                 type: fd
            p2:
                 size_MiB: 2000
                 type: 82
                 format: swap
            p3:
                 type: 83
                 mountpoint: /
                 image: JeOS
            p4:
                 size_MiB: 2000
                 type: 83
                 format: btrfs
                 mountpoint: /data
                 luks_pass: 1234


    md0:
        type: RAID
        level: 1
        devices:
            - disk1p1
            - missing

        disklabel: msdos
        partitions:
            p1:
                type: 82
                format: ext4
                mountpoint: /data2

