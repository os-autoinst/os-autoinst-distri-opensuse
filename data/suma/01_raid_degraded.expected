NAME          MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINT
sda             8:0    0    20G  0 disk  
├─sda1          8:1    0     2G  0 part  
│ └─md0         9:0    0     2G  0 raid1 
│   └─md0p1   259:0    0     2G  0 md    /data2
├─sda2          8:2    0     2G  0 part  [SWAP]
├─sda3          8:3    0  14.1G  0 part  /
└─sda4          8:4    0     2G  0 part  
  └─luks_sda4 253:0    0     2G  0 crypt /data
sr0            11:0    1   3.1G  0 rom   
sr1            11:1    1 300.6M  0 rom   
